README.md

```diff
@@ -0,0 +1,17 @@
+# Gilded Rose Inventory System
+
+## Description
+Welcome to the Gilded Rose Inventory System, this system currently keeps track of our various items sell by date and quality.
+
+## Prerequisites
+- Java
+  - To check your local Java version enter `java --version` in the command line.
+    - **If not installed** Follow the instruction details in this link -> [Java](https://java.com/en/download/)
+- Gradle
+  - To check if you have gradle running locally enter `gradle -v` in the command line
```

Comment:
Question: What version of grade is required to run this?

Reply:
Addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/12/commits/5bf0a9e8200d17ab93f51f2c05267b29b89f26f3)